src/services/Models.service.tsx

```diff
@@ -41,7 +41,7 @@ const validateGlbSource = (source: GlbSource): boolean => {
   }
 
   if (typeof source === 'string') {
-    const fileEndExpression = new RegExp(/(.glb|.glb[?].*)$/g);
+    const fileEndExpression = new RegExp(/(.glb|.fbx|.fbx[?].*|.glb[?].*)$/g);
```

Comment:
I think we'll need to support a `fbxModelExpression` for the binary as well.

Reply:
I don't know how to trigger the blob expression I think it's not used, uploaded drag'n'drop uses string still, pointing to file also uses string, any ideas?

Reply:
I think it's this- https://github.com/mrdoob/three.js/blob/bee2b8ace8be7074cf0a34e3c9653eedcb9c512f/examples/js/loaders/FBXLoader.js#L3853

Let it be for now. I think we won't be serving FBX blobs from our services. Until we do no need to headache over this.